packages/db/src/test/compilation.graphql.ts

```diff
@@ -13,7 +14,7 @@ export const AddCompilation = gql`
         compilations: [
           {
             compiler: { name: $compilerName, version: $compilerVersion }
-            sourceMaps: [{ json: $sourceMap }]
+            sourceMaps: [{ bytecode: { id: $bytecodeId }, data: $sourceMap }]
```

Comment:
Just want to make a note here that we should update the data model to reflect this change.